src/testing/fixtures/algorand-fixture.ts

```diff
@@ -92,21 +93,29 @@ export function algorandFixture(fixtureConfig?: AlgorandFixtureConfig, config?:
   const indexer = fixtureConfig?.indexer ?? getAlgoIndexerClient(config.indexerConfig)
   const kmd = fixtureConfig?.kmd ?? getAlgoKmdClient(config.kmdConfig)
   let context: AlgorandTestAutomationContext
+  let algorandClient: AlgorandClient
 
   const beforeEach = async () => {
     Config.configure({ debug: true })
     const transactionLogger = new TransactionLogger()
     const transactionLoggerAlgod = transactionLogger.capture(algod)
+    const acc = await getTestAccount(
+      { initialFunds: fixtureConfig?.testAccountFunding ?? algos(10), suppressLog: true },
+      transactionLoggerAlgod,
+      kmd,
+    )
+    algorandClient = AlgorandClient.fromClients({ algod: transactionLoggerAlgod, indexer, kmd }).withAccount(acc)
+    const testAccount = { ...acc, signer: algorandClient.account.getSigner(acc.addr) }
     context = {
       algod: transactionLoggerAlgod,
       indexer: indexer,
       kmd: kmd,
-      testAccount: await getTestAccount(
-        { initialFunds: fixtureConfig?.testAccountFunding ?? algos(10), suppressLog: true },
-        transactionLoggerAlgod,
-        kmd,
-      ),
-      generateAccount: (params: GetTestAccountParams) => getTestAccount(params, transactionLoggerAlgod, kmd),
+      testAccount,
+      generateAccount: async (params: GetTestAccountParams) => {
+        const account = await getTestAccount(params, transactionLoggerAlgod, kmd)
+        algorandClient.withAccount(account)
+        return { ...account, signer: algorandClient.account.getSigner(account.addr) }
+      },
```

Comment:
This means generated accounts in testing will automatically be registered with the AlgorandClient that is now also available in the fixture.